### DISCUSSION TITLE
Stuck on older version

### DISCUSSION THREAD

**Discussion by aeroperf** [JanDeDobbeleer/oh-my-posh#1265] [Troubleshoot] [answered]
I just realized I'm on a much older version and cannot seem to move from that. What is the removal process so I can start over?

![image](https://user-images.githubusercontent.com/1622891/142725896-85e842b2-120f-4379-85ba-9634883e18cb.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@aeroperf  you should be able to use `Update-Module oh-my-posh`, that's out of our control. Make sure to restart PowerShell afterwards as well.

**Reply by aeroperf** [reply]
Thank you. I did not realize Powershell has to be restarted to show new version. I'm at the latest.
